variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "project_id" {
+  type        = string
+  description = "Id of GCP project."
+}
+
+variable "network_name" {
+  type        = string
+  description = "Name of VPC network."
+}
+
+variable "subnet_region" {
```

Comment:
The region and zone are injected by the nullstone engine via `GOOGLE_REGION` and `GOOGLE_ZONE`.
You should be able to access the region through the following:
```
data "google_compute_zones" "available" {}

# The region configured on the provider => data.google_compute_zones.region
```

Reply:
Taken care.